README.md

```diff
@@ -512,6 +512,30 @@ new BrowserCheck('browser-check-1', {
 
 😔 sorry, no docs here yet. We are working on getting model right for Check groups.
```

Comment:
Should this be removed yet?

Reply:
I'm expecting some comments from Tim here given my writing skills are bare minimum :)

Reply:
I will review and maybe push some changes to this branch

Reply:
Go for it